src/rtk/features/userInfo/userInfoApi.ts

```diff
@@ -25,18 +31,21 @@ export const userInfoApi = createApi({
     },
   }),
   endpoints: (builder) => ({
-    getUserInfo: builder.query<UserInfordata, void>({
+    getUserInfo: builder.query<UserInfoData, void>({
       query: () => 'profile',
       keepUnusedDataFor: 300,
       transformResponse: (response: UserInfoApiResponse) => {
         return { name: response.party.name };
       },
     }),
-    getReportee: builder.query<UserInfordata, void>({
+    getReportee: builder.query<ReporteeInfoData, void>({
       query: () => `reporteelist/${getCookie('AltinnPartyId')}`,
       keepUnusedDataFor: 300,
       transformResponse: (response: ReporteeApiResponse) => {
-        return { name: response.name };
+        return {
+          name: response.name,
+          orgNumber: response.organizationNumber,
+        };
```

Comment:
Also nitpic, but I don't think you need to transform the response when you just want to return it as the object specified in the query definition.
This queary will automatically return a ReporteeInfoData object as is.

TransformResponse is for when you wish to return something else, or process the data before returning it. (See singleRightApi for examples without transformResponse)

Reply:
Great tip 👍🏼 . I removed the "transformResponse" ang changed the code to use organizationNumber instead of org-number!